pkgs/by-name/si/simdutf/package.nix

```
@@ -0,0 +1,30 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "simdutf";
+  version = "4.0.5";
+
+  src = fetchFromGitHub {
+    owner = "simdutf";
+    repo = "simdutf";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-HNTVo/uB7UTCy5VVdmf6vka9T+htra7Vk7NF4hByGP4=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  meta = with lib; {
+    description = "Unicode routines (UTF8, UTF16, UTF32): billions of characters per second";
```

```suggestion
    description = "Unicode validation and transcoding at billions of characters per second";
```

A better description imnsho.